.percy.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": 1,
+  "snapshot": {
+    "widths": [375, 720, 1280, 1440, 1920],
+    "min-height": 1024,
+    "percy-css": "iframe { display: none; }"
+  },
+  "static-snapshots": {
+    "path": "_site/",
+    "base-url": "/blog/",
+    "snapshot-files": "**/*.html",
+    "ignore-files": "**/*.htm"
+  },
+  "agent": {
+    "asset-discovery": {
+      "allowed-hostnames": [
+        "https://1q5n8517v76m59e8m43p09gr-wpengine.netdna-ssl.com/*",
+        "https://3fxtqy18kygf3on3bu39kh93-wpengine.netdna-ssl.com/*"
+      ],
+      "request-headers": {
```

Comment:
Where is this parameter used?

Reply:
request-headers were not used in the context of the project. I used it as they were from the percy docs. I will remove them.